src/Http/Middleware/HttpsEnforcerMiddleware.php

```diff
@@ -80,6 +82,10 @@ public function __construct(array $config = [])
      */
     public function process(ServerRequestInterface $request, RequestHandlerInterface $handler): ResponseInterface
     {
+        if (is_array($this->config['trustedProxies'])) {
+            $request->setTrustedProxies($this->config['trustedProxies']);
```

Comment:
You'll have to wrap this in a `$request instanceof ServerRequest)` check to naked the static analyzers happy.